<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddConstraintStudentsMaxDaysPerWeekForm_template</class>
 <widget class="QDialog" name="AddConstraintStudentsMaxDaysPerWeekForm_template">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string extracomment="This is the title of the dialog to add a new constraint of this type">Add students max days per week</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <spacer name="spacer1_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="maxDaysTextLabel">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>Max days per week</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="maxDaysSpinBox"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="spacer1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="groupTextLabel">
       <property name="text">
        <string>Group:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="essentialRadioButton">
       <property name="text">
        <string>Essential</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="importantRadioButton">
       <property name="text">
        <string>Important</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="desirableRadioButton">
       <property name="text">
        <string>Desirable</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="weightTextLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>Weight percentage
(necessary: 100%)</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="weightLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string notr="true">100</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="addConstraintPushButton">
       <property name="text">
        <string>Add constraint</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closePushButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
